### ISSUE TITLE
Works only once

### ISSUE THREAD

**Issue by s-t-a-t-e** [sonburn/symbol-swapper#45]
I'm updating an old Sketch file to work with a library. Symbol Swapper looked good.
I'm having a problem with an install of the plugin with Sketch 63.1
When I selected a group of symbols in the original file, and use the function "Symbol Swap Selection", it only worked once. A second time, the plugin didn't respond. No alert. Nothing.
Any ideas?

**Comment by sonburn** [comment]
@s-t-a-t-e there was an issue with the last version, which I finally got a chance to resolve. Could you please try version 0.21 which I just published and let me know if you are experiencing the same problem?

**Comment by sonburn** [comment]
The issue had to do with missing and disabled libraries not being removed from the list of available libraries, this is why the results were offset at times, or other times the plugin would fail. Version 0.21 was a step in the right direction but there was still another issue which I have resolved in 0.22.
